cfg/environment.go

```diff
@@ -121,3 +122,11 @@ func (c *envConfig) PrintDeadline() time.Duration {
 
 	return printDeadline
 }
+
+func (c *envConfig) S3Bucket() string {
+	if v := os.Getenv("PDF_RENDERER_S3_BUCKET"); len(v) != 0 {
+		return  v
+	}
+	log.Fatal("Environment variable PDF_RENDERER_S3_BUCKET must be set.")
```

Comment:
If this is fatal, we should check that it's a valid value before starting the web server and exit if it's  invalid.

Reply:
"Valid value" being a valid s3 name? `log.Fatal` should call `os.Exit(1)` under the hood, did you want this check done before the server starts? If so, any objection to tossing this check inside of the cfg package init?

Reply:
No objection.

Reply:
It should just have a non-empty, non-whitespace value.

Reply:
I've put the `log.Fatal` inside of the init.